Erro panda

[21:53:48 FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.IndexOutOfBoundsException: Index 49 out of bounds for length 45
        at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
        at java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
        at net.minecraft.server.v1_8_R3.SystemUtils.a(SystemUtils.java:12) ~[patched_1.8.8.jar:git-PandaSpigot-83]
        at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:880) ~[patched_1.8.8.jar:git-PandaSpigot-83]
        at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:412) ~[patched_1.8.8.jar:git-PandaSpigot-83]
        at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:814) ~[patched_1.8.8.jar:git-PandaSpigot-83]
        at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:647) ~[patched_1.8.8.jar:git-PandaSpigot-83]
        at net.minecraft.server.v1_8_R3.MinecraftServer.lambda$spin$0(MinecraftServer.java:133) ~[patched_1.8.8.jar:git-PandaSpigot-83]
        at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.IndexOutOfBoundsException: Index 49 out of bounds for length 45
        at jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64) ~[?:?]
        at jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70) ~[?:?]
        at jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266) ~[?:?]
        at java.util.Objects.checkIndex(Objects.java:359) ~[?:?]
        at java.util.ArrayList.get(ArrayList.java:427) ~[?:?]
        at net.minecraft.server.v1_8_R3.Container.getSlot(Container.java:109) ~[patched_1.8.8.jar:git-PandaSpigot-83]
        at net.minecraft.server.v1_8_R3.PlayerConnection.a(PlayerConnection.java:1732) ~[patched_1.8.8.jar:git-PandaSpigot-83]
        at net.minecraft.server.v1_8_R3.PacketPlayInWindowClick.a(PacketPlayInWindowClick.java:17) ~[patched_1.8.8.jar:git-PandaSpigot-83]
        at net.minecraft.server.v1_8_R3.PacketPlayInWindowClick.a(PacketPlayInWindowClick.java:5) ~[patched_1.8.8.jar:git-PandaSpigot-83]
        at net.minecraft.server.v1_8_R3.PlayerConnectionUtils$1.run(PlayerConnectionUtils.java:9) ~[patched_1.8.8.jar:git-PandaSpigot-83]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539) ~[?:?]
        at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
        at net.minecraft.server.v1_8_R3.SystemUtils.a(SystemUtils.java:11) ~[patched_1.8.8.jar:git-PandaSpigot-83]
        ... 6 more

--- a/patches/server/0070-Fix-AIOOBE-in-inventory-handling.patch
+++ b/patches/server/0070-Fix-AIOOBE-in-inventory-handling.patch
@@ -5,15 +5,15 @@ Subject: [PATCH] Fix AIOOBE in inventory handling
 
 
 diff --git a/src/main/java/net/minecraft/server/PlayerConnection.java b/src/main/java/net/minecraft/server/PlayerConnection.java
-index 15f7c315df23eac08e12ce752b1b1bb23fd25aba..14afdc7cbb88150f4711811a06fa149c36cec8b9 100644
+index 15f7c315df23eac08e12ce752b1b1bb23fd25aba..557bb0d99b227ed0a70b1855f9ddab88f94dc134 100644
 --- a/src/main/java/net/minecraft/server/PlayerConnection.java
 +++ b/src/main/java/net/minecraft/server/PlayerConnection.java
-@@ -1505,7 +1505,7 @@ public class PlayerConnection implements PacketListenerPlayIn, IUpdatePlayerList
-                     }
-                     if (packetplayinwindowclick.c() == 0 || packetplayinwindowclick.c() == 1) {
-                         action = InventoryAction.NOTHING; // Don't want to repeat ourselves
+@@ -1600,7 +1600,7 @@ public class PlayerConnection implements PacketListenerPlayIn, IUpdatePlayerList
+                 } else if (packetplayinwindowclick.f() == 3) {
+                     if (packetplayinwindowclick.c() == 2) {
+                         click = ClickType.MIDDLE;
 -                        if (packetplayinwindowclick.b() == -999) {
 +                        if (packetplayinwindowclick.b() < 0) { // PandaSpigot - Paper Issue GH-404
-                             if (player.inventory.getCarried() != null) {
-                                 action = packetplayinwindowclick.c() == 0 ? InventoryAction.DROP_ALL_CURSOR : InventoryAction.DROP_ONE_CURSOR;
-                             }
+                             action = InventoryAction.NOTHING;
+                         } else {
+                             Slot slot = this.player.activeContainer.getSlot(packetplayinwindowclick.b());

